AEPServices/Sources/ui/MessageSettings.swift

```diff
@@ -0,0 +1,77 @@
+/*
+ Copyright 2021 Adobe. All rights reserved.
+ This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License. You may obtain a copy
+ of the License at http://www.apache.org/licenses/LICENSE-2.0
+ 
+ Unless required by applicable law or agreed to in writing, software distributed under
+ the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ OF ANY KIND, either express or implied. See the License for the specific language
+ governing permissions and limitations under the License.
+ */
+
+import Foundation
+import UIKit
+
+@objc(AEPMessageSettings)
+public class MessageSettings: NSObject {
+    public let parent: Any?
```

Comment:
I am not familiar with the use case here but does `parent` change based on when the message is displayed?

Reply:
no, the parent is the class that owns the Message that's being created - it's just a dependency injection to allow more control for the caller